identity/src/ecdsa.rs

```diff
@@ -258,4 +295,11 @@ mod tests {
         let invalid_msg = "h3ll0 w0rld".as_bytes();
         assert!(!pk.verify(invalid_msg, &sig));
     }
+
+    #[test]
+    fn serialize_deserialize() {
```

Comment:
I've added some tests based on test vectors (https://github.com/libp2p/specs/pull/537) in [`92f6914` (#3681)](https://github.com/libp2p/rust-libp2p/pull/3681/commits/92f6914bf5c6c8dc12146c64c967f407000354a0) so we can delete this again.